src/web/components/Core/Form.spec.tsx

```diff
@@ -23,27 +22,34 @@ describe('Form Component', () => {
 
     const textInput = screen.getByTestId('text-input');
 
-    await act(async () => {
-      await userEvent.type(textInput, 'New value');
-      await userEvent.click(screen.getByRole('radio', { name: 'No' }));
-      await userEvent.click(screen.getByRole('checkbox', { name: 'Checkbox 1' }));
+    userEvent.type(textInput, 'New value');
```

Comment:
If you just await the user events, you shouldn't then need to use all the waitFor blocks.

Reply:
the await will definitely cause the following error. I've spent hours debugging this last night and found the only to resolve this errors was remove the `await` and use waitFor to make sure all updates are done.
 
```
console.error
    Warning: An update to null inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
```